invenio_users_resources/records/api.py

```diff
@@ -156,6 +156,17 @@ def commit(self):
         self.model.update(data)
         return self
 
+    @classmethod
+    def deactivate(cls, id_):
```

Comment:
Added to invalidate the identity immediately, otherwise the users are deactivated and can still do things in the platform until they're logged off.